cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApiImpl.java

```diff
@@ -435,6 +436,10 @@ private void httpExceptionHandler(Promise<? extends CruiseControlResponse> resul
             // Vert.x throws a NoStackTraceTimeoutException (inherits from TimeoutException) when the request times out
             // so we catch and raise a TimeoutException instead
             result.fail(new TimeoutException(t.getMessage()));
+        } else if (t instanceof NoRouteToHostException) {
+            // Netty throws a AnnotatedNoRouteToHostException when it cannot resolve the host
```

Comment:
You talk about `AnnotatedNoRouteToHostException` buit you do not seem to use it. So ids the code wrong? Or the comment wrong?

Reply:
Hi Jakub, Actually we do get a `AnnotatedNoRouteToHostException` but we catch `NoRouteToHostException` instead since `AnnotatedNoRouteToHostException` (inherits from NoRouteToHostException)

Reply:
So the comment is wrong?